tests/unit/phpDocumentor/Guides/DocumentTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * phpDocumentor
+ *
+ * PHP Version 5.5
+ *
+ * @copyright 2010-2015 Mike van Riel / Naenius (http://www.naenius.com)
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor\Guides;
+
+use phpDocumentor\Path;
+
+/**
+ * Test case for Document
+ * @coversDefaultClass phpDocumentor\Guides\Document
+ */
+class DocumentTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @covers ::__construct
+     * @covers ::getTitle
+     * @covers ::getPath
+     * @covers ::getContent
+     */
+    public function testGetters()
+    {
+        $document = new DummyDocument(new Path('my/Path'), 'SomeTitle', 'MyContent');
```

Comment:
You can also create a partial mock using mockery where you only mock the getContentType method; that will enable you to omit the DummyDocument. I don't mind either approach so if you prefer this then I am fine with it, I didn't know if you knew about the partial mock technique for testing abstract classes